source/contribute/maintainer.rst

```diff
@@ -22,6 +22,14 @@ following:
 -  Creating release branches.
 -  Tagging released versions within **master** and release branches.
 
+Automation
+----------
+
+Each of the following steps are documented for humans to perform them
+manually, but maintainers and contributors are highly encouraged to try
+and automate them away into tooling that is commonly available across
+the organisation projects, as well as the community at large.
```

Comment:
We also talked about automating PR merges, after passed builds and approvals (and i.e. no `blocking` label).
Ideally we should also guard automatic merges by [`Roave/BackwardCompatibilityCheck`](https://github.com/Roave/BackwardCompatibilityCheck) that would block auto-merge (ideally as a separate CI integration, not part of the Travis build directly).

Reply:
Yrah, Roave BC is a bit too strict, but we can add our (laxer) rules to CI. Also, I need to make it faster before it is introduced.